### DISCUSSION TITLE
Strange "Custom Properties" displays in sidebars

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#88465] [Repositories]
### Select Topic Area

Bug

### Body

For several repos (but not all)...

![image](https://github.com/community/community/assets/10906554/175ef62c-6211-4e85-9839-1942c86bf388)

![image](https://github.com/community/community/assets/10906554/f5dae898-739a-4770-8259-e5949cae523c)

... a strange, boilerplate-looking "Custom Properties" property name appears in sidebar (instead of an expected actual custom property name)

**Comment by sameera222** [comment]
It could be related to the GitHub interface or how custom properties are being handled in the particular repositories. Here are some steps you can take to troubleshoot and potentially fix the issue:    

Check for GitHub Outages:
Visit GitHub's status page (https://www.githubstatus.com/) to check if there are any ongoing issues or outages that might be affecting the display of custom properties.

Disable Browser Extensions:
Temporarily disable browser extensions to rule out any interference. Some extensions may modify the page's stylesheets and cause unexpected behavior.

Review GitHub Actions or Workflows:
If your repositories are using GitHub Actions or workflows, check for any scripts or configurations that might be affecting the display in the GitHub interface. Review the GitHub Actions logs for any relevant information.

**Reply by adamlui** [reply]
Your solutions are nonsensical as it pertains to thtis bug unfortunately
